deploy/docker/Dockerfile

```diff
@@ -1,13 +1,17 @@
 # Docker image for the primary terria map application server
-FROM node:8
+FROM osgeo/gdal:alpine-small-3.1.0 as build
 
-RUN apt-get update && apt-get install -y gdal-bin
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh npm
+RUN mkdir -p /etc/config/client
+WORKDIR /usr/src/app
 
-RUN mkdir -p /usr/src/app && mkdir -p /etc/config/client
-WORKDIR /usr/src/app/component
-COPY . /usr/src/app
-RUN rm wwwroot/config.json && ln -s /etc/config/client/config.json wwwroot/config.json
+COPY ./package.json ./
+RUN npm install
 
-EXPOSE 3001
```

Comment:
Why was this removed? I'm a bit of a Docker novice, and I checked the EXPOSE docs, but I'm still a bit confused as it what it actually does. Would you mind explaining so I can learn 😄 ?

Reply:
Hi @KeyboardSounds sorry, EXPOSE is usually used for documentation porpouse as you can port forward any port you want in theory,. In my case I define the port we want to expose in our helm chart and it is not necessarly the 3001 as we run the npm command with the flag --port and <port number>. But we can definitelly add the EXPOSE in dockerfile,